docs/api-reference/projects.md

```
@@ -0,0 +1,326 @@
+
+| Status Code | Description |
+|-------------|-------------|
+| **204** | Successful Response |
+| **422** | Validation Error |
+
+**Response Body** (application/json):
+
+| Name | Type | Description |
+|------|------|-------------|
+| `detail` | array[object] | - |
+**`detail`** — Array of `object`
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)